### PR TITLE
libmms: fix build for Linux

### PR DESCRIPTION
**Pull request by cho-m**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

https://github.com/Homebrew/homebrew-core/runs/3041576152?check_suite_focus=true

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

**Comment by danielnachun** [comment]
Let's add the license:
`LGPL-2.1-or-later`

File: Formula/libmms.rb [hunk 1]
```diff
@@ -20,7 +21,7 @@ class Libmms < Formula
   depends_on "glib"
 
   def install
-    ENV.append "LDFLAGS", "-liconv"
+    on_macos { ENV.append "LDFLAGS", "-liconv" }
```
Comment (SMillerDev) [inline Formula/libmms.rb:24]: Doesn't this mean it `uses_from_macos`?

Comment (cho-m) [reply Formula/libmms.rb:24]: `iconv` should be in `glibc` on Linux, so no extra dependency.

macOS just needs the extra flag to find library.

**Review by danielnachun — APPROVED** [review]

**Comment by chenrui333** [comment]
Dispatched, https://github.com/Homebrew/homebrew-core/runs/3062595721

**Pull request merged**